source/deployment/ldap-group-constrained-team-channel.rst

```diff
@@ -0,0 +1,61 @@
+.. _ldap-group-constrained-team-channel:
+
+Using AD/LDAP Synchronized Groups to Manage Team or Private Channel Membership
+===============================================================
+Mattermost groups created with synchronized AD/LDAP groups can be used to manage the membership of private teams and private channels. When a team or private channel is managed by synchronized groups, users will be added and removed based on their membership to the synchronized AD/LDAP group.  
+
+Users outside of the group are restricted from being invited through a team invite link or an email invite on teams that are managed by synchronized groups. Similarly, users outside of synchronized groups are restricted from being invited through a mention, the /invite slash command or directly with “add members” in channels.  Users, however, can remove themselves from teams and private channels managed by synchronized groups.  
```

Comment:
Wondering if this could be a bullet list for easier readability

Reply:
Updated.